Dockerfile

```diff
@@ -1,26 +1,40 @@
 FROM --platform=linux/amd64 debian:11.5
 
-RUN apt-get update && \
-    apt-get -y install gnupg jq ca-certificates cmake curl g++ gcc git libx11-dev libffi-dev libnss3-tools locales make libarchive-tools latexmk texlive-latex-recommended texlive-latex-extra\
-    texlive texlive-xetex texlive-lang-european texlive-fonts-recommended texlive-fonts-extra inkscape netcat-traditional ruby ruby-dev sudo libsystemd-dev golang vim cups-ipp-utils binutils && \
-    curl 'https://dl-ssl.google.com/linux/linux_signing_key.pub' | apt-key add - && \
+RUN apt-get update
+
+RUN apt-get -y install gnupg jq ca-certificates cmake curl g++ gcc git libx11-dev libffi-dev libnss3-tools locales make libarchive-tools latexmk texlive-latex-recommended texlive-latex-extra \
+    texlive texlive-xetex texlive-lang-european texlive-fonts-recommended texlive-fonts-extra inkscape netcat-traditional ruby ruby-dev sudo libsystemd-dev golang vim cups-ipp-utils binutils
```

Comment:
Voikohan tässä käydä nyt niin, että jos tätä tokaa run komentoa muokkaa, niin se ei enää aja apt-get updatea uudestaan kun se on cachettu? Ongelmia mitä siitä vois tulla, niin asennetaan joku vanha paketti tai jotain pakettia ei löydy (varmaan feilais siihen)

Reply:
Mä näkisin et toi updaten tulos ei muutu vaikka tonne install loitsuun lisäis mitä -> updaten kakuttaminen pitäs sit bustata tarkoituksella jollain muulla tavalla.

Reply:
Tarkennan ☝️ että jos noi on samassa runissa, se updaten tulos toki päivittyy siinä mielessä että se ajetaan uudelleen jos lisätään listaan depsu, mutta se depsu itse ei vaikuta siihen updaten tulokseen :D

Reply:
Joo noinhan se on. Aika epätodennäköinen toi mun skenaario on ongelmana, ja tässä varmaan olis myös hyvä jos vaikka aina välillä buildattais se pohjaimage uudestaan myös